File: packages/sdk/dotnet/Client/NotFoundException.cs
namespace Ihui.AI;

/// <summary>
/// 404 资源不存在异常。
/// </summary>
public class NotFoundException : SdkException
{
    /// <summary>构造 NotFoundException。</summary>
    /// <param name="status">HTTP 状态码(应为 404)</param>
    /// <param name="code">错误码</param>
    /// <param name="message">错误消息</param>
    /// <param name="details">错误详情</param>
    public NotFoundException(int status, string? code, string message, object? details = null)
        : base(status, code, message, details)
    {
    }
}

File: packages/sdk/java/src/main/java/com/ihui/ai/sdk/NotFoundException.java
package com.ihui.ai.sdk;

/**
 * 404 资源不存在异常。
 */
public class NotFoundException extends SdkException {

    private static final long serialVersionUID = 1L;

    /**
     * 构造 NotFoundException。
     *
     * @param status  HTTP 状态码(应为 404)
     * @param code    错误码
     * @param message 错误消息
     * @param details 错误详情
     */
    public NotFoundException(int status, String code, String message, Object details) {
        super(status, code, message, details);
    }
}
